### PR TITLE
Ops 2481 initial data

### PR DESCRIPTION
**Pull request by johndeange**
## What changed

Provide initial minimal data set for first production release.
Use shell scripts to load test and initial data from Docker.

## Issue

https://github.com/HHS/OPRE-OPS/issues/2481

## How to test

1. Start the application with the command: `docker compose --profile data-initial up --build`
2. Load the admin user in the DB:
```
INSERT INTO ops.ops_user (id, oidc_id, hhs_id, email, first_name, last_name, division, status, created_by, updated_by, created_on, updated_on) VALUES (521, '00000000-0000-1111-a111-000000000019', null, 'user.demo@email.com', 'User', 'Demo', 3, 'ACTIVE', null, null, '2024-07-03 15:51:30.798499', '2024-07-03 15:51:30.798499');

INSERT INTO ops.user_role (user_id, role_id, created_by, updated_by, created_on, updated_on) VALUES (521, 5, null, null, '2024-07-03 18:39:23.755454', '2024-07-03 18:39:23.755454');
```
3. Login as a regular OPS user and interact with a few of the features in the application.

## Definition of Done Checklist
- [x] OESA: Code refactored for clarity
- [x] OESA: Dependency rules followed
- [x] Automated unit tests updated and passed
- [x] Automated integration tests updated and passed
- [x] Automated quality tests updated and passed
- [x] Automated load tests updated and passed
- [x] Automated a11y tests updated and passed
- [x] Automated security tests updated and passed
- [x] 90%+ Code coverage achieved
- [x] Form validations updated

## Links

_If relevant, e.g. for a link to a piece of markdown documentation_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,13 @@ To run the application using the production server configuration...
 docker compose up db data-import backend frontend-static --build
 ````
 
+To run the application using the minimal initial data set...
+
+```shell
+  docker compose --profile data-initial up --build
```
Comment (fpigeonjr) [inline README.md:62]: 🤩

**Review by fpigeonjr — COMMENTED** [review]
LGTM but will defer to other devs

**Review by stevtek — APPROVED** [review]

File: frontend/src/components/Auth/MultiAuthSection.jsx [hunk 2]
```diff
@@ -111,7 +111,7 @@ const MultiAuthSection = () => {
                         You can access your account by signing in with one of the options below.
                     </p>
                 </div>
-                {window.location.href.includes("localhost")  && ( // login.gov is only configured to work locally at the moment
+                {window.location.href.includes("localhost") && ( // login.gov is only configured to work locally at the moment
```
Comment (fpigeonjr) [inline frontend/src/components/Auth/MultiAuthSection.jsx:114]: wonder if we could Vite's `import.meta.env.BASE_URL` 

something like this: 

```jsx
{import.meta.env.BASE_URL.includes('localhost') && (
  // ...rest of code
)}
```

Comment (jonnalley) [reply frontend/src/components/Auth/MultiAuthSection.jsx:114]: @fpigeonjr I had looked into that previously but that env variable requires that it be set manually elsewhere and controlled for each environment which was more overhead than I was looking to add at the time based on the need--under the guise of Minimize & Simplify

Comment (johndeange) [reply frontend/src/components/Auth/MultiAuthSection.jsx:114]: From experimentation with it previously, the `BASE_URL` is looking for a path starting from `/` (without the hostname) so doesn't work here.  `import.meta.env.MODE === "development" &&` works though so I will replace it with this.

File: README.md (hunk 1 above)
Comment (fpigeonjr) [reply README.md:62]: adding an alias to my zshell

File: frontend/src/components/Auth/MultiAuthSection.jsx (hunk 2 above)
Comment (fpigeonjr) [reply frontend/src/components/Auth/MultiAuthSection.jsx]: 🤔 @johndeange I thought the idea was to only show this in local development not our development environment in Azure?

**Review by fpigeonjr — APPROVED** [review]
🤩 testing locally and works good. Had to run docker normally first and then spin down and run with the init script.

File: frontend/src/components/Auth/MultiAuthSection.jsx (hunk 2 above)
Comment (johndeange) [reply frontend/src/components/Auth/MultiAuthSection.jsx]: Yes my change accomplishes this: `MODE === "development"` is the default mode for running out of the vite development server and so the button will only be shown in this environment.

Some more background: `MODE === "production"` is when we are running in production. `import.meta.env.DEV` is the opposite of `import.meta.env.PROD` but not equal to `MODE === "development"`

**Pull request merged**